frontend/src/Layout/Router.jsx

```
@@ -0,0 +1,71 @@
+            },
+            // {
+            //     path: "/messages",
+            //     element: <Message></Message>
+            // },
+        ],
+    },
+    {
+        path: "/studentDashBoard",
+// layout
+        element: <PrivateRoute><StudentDashBoard /></PrivateRoute>,
+        errorElement: <ErrorPage></ErrorPage>,
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)